zcbor/zcbor.py

```diff
@@ -2692,7 +2703,16 @@ def parse_args():
         "-b", "--default-bit-size", required=False, type=int, default=32, choices=[32, 64],
         help="""Default bit size of integers in code. When integers have no explicit bounds,
 assume they have this bit width. Should follow the bit width of the architecture
-the code will be running on."""
+the code will be running on.""")
+    code_parser.add_argument(
+        "-I", "--include-dir", action="append", type=Path, default=list(), nargs="+",
+        help="""Add include directories. #include statements will be relative to the closest dir.
```

Comment:
The closest dir to where? From where this script is called from?

Reply:
to the file being included, so the #include statement is the shortest possible.

Reply:
Added a sentence to the help text about this.